src/components/ViewParty/ViewParty.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useContext } from 'react';
+import { Table, Button, Accordion, Card } from 'react-bootstrap';
+import cookies from 'js-cookie';
+import './ViewParty.css'
+import { UserContext } from '../../UserContext'
+import axios from 'axios';
+
+//temporary metadata until we can pull it from the DB
+const tempPartyList = [
+    {
+      Host: "Thadd",
+      Name: "Detroit LAN",
+      Location: "Detroit",
+      Latitude: 42.331427,
+      Longitude: -83.0457538,
+      Date: "Thu Oct 22 2020 00:58:34 GMT-0400"
+    }
+  ]
+
+
+const ViewParty=()=>{
+
+    return(
+        <div>
+        {tempPartyList.map((p) => (
+          <div 
+            style={{
+              padding: "10px 10px 5px",
+              borderBottom:"2px solid #0C0C0D",
+              backgroundColor: "#35373D"
+            }}>
+            <h4>{p.Name}</h4>
```

Comment:
Change "p" to "props". Do the same for other instances of "p"

Reply:
all instances of p changed to props